packages/office-ui-fabric-react/src/components/Panel/Panel.tsx

```diff
@@ -83,43 +104,27 @@ export class Panel extends BaseComponent<IPanelProps, IPanelState> {
       firstFocusableSelector,
       forceFocusInsideTrap,
       hasCloseButton,
-      headerClassName = '',
       headerText,
       ignoreExternalFocusing,
       isBlocking,
       isLightDismiss,
       layerProps,
-      type
+      type,
+      onRenderNavigation = this._onRenderNavigation,
+      onRenderHeader = this._onRenderHeader,
+      onRenderBody = this._onRenderBody,
+      onRenderFooter = this._onRenderFooter
     } = this.props;
-    let { isOpen, isAnimatingOpen, isAnimatingClose, id } = this.state;
+    let { isOpen, isAnimatingOpen, isAnimatingClose, id, footerIsSticky } = this.state;
```

Comment:
while we're here, there is a bug... if animation doesn't work, we will get stuck in a state. We absolutely rely on animations to work here (which means we have an implicit reliance on fabric-core classes existing, which is flimsy!)

I want to say that we use setTimeout for controlling the animation durations rather than animation start/end events.